testing/tests/DevExpress.ui.events/click.tests.js

```diff
@@ -58,14 +58,19 @@ QUnit.test("event triggers", function(assert) {
 });
 
 QUnit.test("event args", function(assert) {
+    var fields = ["altKey", "button", "cancelable", "clientX", "clientY",
+        "ctrlKey", "currentTarget", "data", "delegateTarget",
+        "isDefaultPrevented", "metaKey", "originalEvent",
+        "pageX", "pageY", "relatedTarget", "screenX", "screenY", "shiftKey",
+        "target", "timeStamp", "type", "view", "which"];
+
+
+    if(!QUnit.urlParams["nojquery"]) {
```

Comment:
@San4es, please, take your attention to this part of code

Reply:
@nightskylark 

It seems, that issue related to `nativePointerMock` implementation:
https://github.com/DevExpress/DevExtreme/blob/bd9e5a18dbd7360328b161f2b9267674bb896a08/testing/helpers/nativePointerMock.js#L716-L740

I suggest just remove this block from this test:

```
if(!QUnit.urlParams["nojquery"]) {
    fields = fields.concat(["bubbles", "buttons", "eventPhase", "handleObj", "offsetX", "offsetY", "toElement"]);
 }
```
because it has no semantic value for this test scenario - all available event properties will be copied.